packages/augur-ui/src/modules/auth/actions/login-with-torus.ts

```diff
@@ -72,13 +72,19 @@ export const loginWithTorus = () => async (
       throw error;
     }
 
-    try {
-      const userInfo = await torus.getUserInfo();
-      accountObject.meta.email = userInfo.email;
-      accountObject.meta.profileImage = userInfo.profileImage;
-      dispatch(updateSdk(accountObject, undefined));
-    } catch (error) {
+    // Temporary workaround - Torus userInfo modal is blocked on safari
+    const isSafari = !!navigator.userAgent.match(/Version\/[\d\.]+.*Safari/);
```

Comment:
I think there is another place in UI code we are determining if user is using safari. If so, Can you consolidate and make a utility method.